plugins/accelerated-peft/src/fms_acceleration_peft/framework_plugin_autogptq.py

```diff
@@ -275,9 +275,6 @@ def augmentation(
 
         # some assertions
         assert peft_config is not None, "need peft_config to install PEFT adapters"
-        assert (
```

Comment:
ok so, we possibly will need to move this to foak plugin.  my guess is that the only thing we need to enforce is

```python
if model.dtype == torch.float16`:
    assert train_args.fp16
```

Reply:
test

dtype | mp | outcome
--|--|--
float16 | Y |
foat16 | N|
bfloat16 | Y 
bfloat16 | N